docs/organizer-guide/meetups/covid-19-meetups.rst

```diff
@@ -0,0 +1,12 @@
+Meetups and COVID-19
+====================
+
+With the recent announcement about the Write The Docs NA (Portland) conference, we wanted to provide guidance for our community of meetups.
```

Comment:
s/NA (Portland)/Portland/

Reply:
> Should we include a short list of suggestions for remote meetups here? Email book club, remote speaker, collaborative writing....

I'm tempted to push such questions to #wtd-meetup-remote

Reply:
PS, I've warned one of the leaders of #wtd-meetup-remote, and await his concurrence.

Reply:
I have concurrence to point to #wtd-meetup-remote for advice on howto remote meetup